panel/styles/modules/output.styl

```diff
@@ -17,7 +17,7 @@
   .CodeMirror
     height 100%
     box-shadow none
-    -webkit-box-shadow none
+    box-shadow none
```

Comment:
This line can go; no need for a prefixed version and unprefixed is on the previous line